milestone-3/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+node_modules/
```

Comment:
the gitignore for some reason isn't working, as 528 files were added to this PR. Not. a huge deal tho.